### DISCUSSION TITLE
Move hamburger menu to right side

### DISCUSSION THREAD

**Discussion by royfrancis** [quarto-dev/quarto-cli#9135] [Q&A]
### Description

Is there an option to move the hamburger menu to the right side? I think it pushes the logo far out especially on phone screens.

![Screenshot 2024-03-19 at 09 51 00](https://github.com/quarto-dev/quarto-cli/assets/5083500/9ad0f076-1673-45dc-9178-8caced8ad3e8)

`quarto 1.4.549`

**Comment by mcanouil** [comment]
It's not an option, but as for any HTML, if you want to tweak spacing or nearly anything you can always use CSS, especially for opinionated design choices.
Use your browser inspect/developer mode and create a (S)CSS rule to change the aspect.
